src/command_modules/azure-cli-eventgrid/setup.py

```diff
@@ -32,7 +32,8 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'six'
+    'six',
```

Comment:
@derekbekoe is this needed?

Reply:
Since that dependency is already in `azure-cli-core`, it's not needed but doesn't hurt anything either.